examples/evaluations/litellm_vectorizer/compose.yaml

```diff
@@ -0,0 +1,24 @@
+name: pgai 
+services:
+  db:
+    image: timescale/timescaledb-ha:pg17
+    environment:
+      POSTGRES_PASSWORD: postgres
+      OPENAI_API_KEY: ""
+      COHERE_API_KEY: ""
+      MISTRAL_API_KEY: ""
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./data:/var/lib/postgresql/data
+  vectorizer-worker:
+    image: timescale/pgai-vectorizer-worker:latest
+    environment:
+      PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
+      OPENAI_API_KEY: ""
+      COHERE_API_KEY: ""
+      MISTRAL_API_KEY: ""
```

Comment:
The API keys are now actually passed forward from the db to the worker so we don't have to set it twice. I'm actually not 100% sure this works with LiteLLM but I hope it does.